### DISCUSSION TITLE
[docs] nuxi init flags incomplete

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [nuxt/nuxt#27035] [Questions] [answered]
https://nuxt.com/docs/api/commands/init

Is not showing `--no-install` and `--packageManager`.

**Comment by TheAlexLichter** [comment]
PR welcome!

**Comment by angelhdzmultimedia** [comment]
Went to edit the file but I'm confused:

```
 // Install project dependencies
    // or skip installation based on the '--no-install' flag
    if (ctx.args.install === false) {
      consola.info('Skipping install dependencies step.')
    } else {
      consola.start('Installing dependencies...')
 ```
      
In the main branch of the code the arg is named "ctx.args.install" instead of "ctx.args.noInstall" or "ctx.args['no-install']".

Description says `Skip install` but the default value is `true`...

I just created a new nuxt project with `pnpx nuxi@latest init test --no-install` and it indeed created it without installing deps.

And of course, `pnpx nuxi@latest init --help` shows `--no-install`.
And calling `pnpx nuxi@latest init test` without flags, installs dependencies as expected. So this must be a mistake?

Or is a obscure place where `--no-install` is being checked? I looked at all the files like run, main, etc.

### Also
`pnpx nuxi@latest init --help` doesn't show `dir` flag, but the code in main branch does...

Maybe @pi0 can help with this info so that I can go ahead and make the pr.

**Reply by TheAlexLichter** [reply, marked as answer]
@angelhdzmultimedia No, that is based on the `install` flag. The underlying "citty" package handles the "no" prefix ☺️ https://github.com/unjs/citty/blob/0ea32b6c033839957744938f51dcf5b867d425e9/src/usage.ts#L49-L65
